tests/test_standard_tags.py

```diff
@@ -181,20 +181,18 @@ def test_client_ip(self):
 
         def validator(span):
             meta = span.get("meta", {})
-            if "http.client_ip" not in meta:
-                raise Exception("missing http.client_ip tag")
+            assert "http.client_ip" not in meta, "missing http.client_ip tag"
```

Comment:
be careful, `if condition then raise` <=> `assert not condition`. Same for other modifs

```
assert "http.client_ip" in meta, "missing http.client_ip tag"
```

Reply:
yes... sorry, fixed